opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/AbstractJettyTest.java

```diff
@@ -114,6 +119,34 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         }
     }
 
+    public static class AsyncServlet extends HttpServlet {
+
+        public static int ASYNC_SLEEP_TIME = 3000;
```

Comment:
In general, yes. Most of the time, you can find a better solution with a `CountDownLatch`: if you need to delay a processing, you can make it accept a latch, and `await()` on it. Once you are done and the code can be resumed, you count down, and the code will continue executing.

There are two problems with fixed times: they are usually waaaay higher than what's normally needed, so, they delay the build. A 3s wait might not sound much (it is), and if you keep adding async tests, each adding 3s to the build time, you end up having to wait an eternity very soon :)
The other problem is that there *will* be executions where the 3s will not be enough, but 3.1s will. So, your build will fail, without a good reason to do so.